src/lib/annotations/annotatorUtil.js

```diff
@@ -41,7 +41,7 @@ export function findClosestElWithClass(element, className) {
 */
 export function getPageInfo(element) {
     const pageEl = findClosestElWithClass(element, 'page') || null;
-    let page = -1;
+    let page = 1;
```

Comment:
Make sure that this works. I know we do some explicit checks for `page === -1` so that we can check where events occur on the page.